File: app/src/main/java/mb00/android/codehub/ui/about/inject/AboutActivityModule.java
package mb00.android.codehub.ui.about.inject;

import dagger.Module;
import dagger.Provides;
import mb00.android.codehub.ui.about.viewmodel.AboutViewModel;


@Module
public class AboutActivityModule {

    @Provides
    AboutViewModel provideAboutViewModel() {
        return new AboutViewModel();
    }

}

File: app/src/main/java/mb00/android/codehub/ui/about/inject/AboutActivityModule.kt
package mb00.android.codehub.ui.about.inject

import dagger.Module
import dagger.Provides
import mb00.android.codehub.ui.about.viewmodel.AboutViewModel


@Module
class AboutActivityModule {

    @Provides
    internal fun provideAboutViewModel(): AboutViewModel {
        return AboutViewModel()
    }

}
